.travis.yml

```diff
@@ -145,12 +145,21 @@ jobs:
         - zcat xtensa-esp32-elf-linux64-1.22.0-80-g6c4433a-5.2.0.tar.gz | tar x
         - export PATH=$(pwd)/xtensa-esp32-elf/bin:$PATH
         - git clone https://github.com/espressif/esp-idf.git
-        - git -C esp-idf checkout $(grep "ESPIDF_SUPHASH :=" ports/esp32/Makefile | cut -d " " -f 3)
-        - git -C esp-idf submodule update --init components/json/cJSON components/esp32/lib components/esptool_py/esptool components/expat/expat components/lwip/lwip components/mbedtls/mbedtls components/micro-ecc/micro-ecc components/nghttp/nghttp2
+        - export IDF_PATH=$(pwd)/esp-idf
```

Comment:
Looks like the `sudo pip3 install pyparsing` line above needs to be `sudo pip3 install 'pyparsing<2.4'`

Reply:
Thanks, updated the branch.